packages/fxa-payments-server/server/config/index.js

```diff
@@ -24,6 +24,12 @@ const conf = convict({
       env: 'SUBSCRIPTIONS_STRIPE_TAX_ENABLED',
       format: Boolean,
     },
+    stripeInvoiceImmediately: {
+      default: false,
+      doc: 'Enables immediate invoicing for stripe in all circumstances',
+      env: 'SUBSCRIPTIONS_STRIPE_INVOICE_IMMEDIATELY',
+      format: Boolean
+    }
```

Comment:
Nit: for consistency can we change it to `useStripeInvoiceImmediately` and add a comma after `Boolean`?

Also, should we add the term `upgrade` or `upgrades` somewhere in the `doc` for both? Just so devs know this is specifically targeting upgrades? Or because this is a temp variable and can be used for all immediately invoicing purposes, we should leave it?

Reply:
done, done, and done (naming, comma, docs)